gallia-prehistoire.csl

```diff
@@ -62,8 +61,8 @@
   </macro>
   <macro name="editor">
     <names variable="editor">
-      <name initialize-with="." name-as-sort-order="all"/>
-      <label form="short"/>
+      <name et-al-min="3" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
```

Comment:
The `et-al-min` isn't necessary here; it's inherited from the citation/bibliography. Please remove.